index.js

```diff
@@ -72,9 +72,10 @@
 
  Another option is to call OAuth.correctTimestamp with a Unix timestamp.
  */
-var CryptoJS = require('crypto-js');
+var CryptoJS = require('crypto-js'),
+    OAuth;
+const STRING = 'string';
 
-var OAuth;
 if (OAuth == null) OAuth = {};
```

Comment:
When would this condition be satisfied?

Reply:
Umm.... always. Not sure why was this added in the first place. 
We could have initialized it with an empty object. Maybe we just moved this code from somewhere directly without modifying.

Reply:
That safeguard is present to ensure that if you include this file in the browser, you don't initialize the object twice.